Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:20.04
+
+LABEL desc="Docker container for building and testing plugin-utilities"
+
+# Install OpenJDK11
+RUN apt-get update && apt-get install -y openjdk-11-jdk
+
+# Install Git
+RUN apt -y install git-all
+
+# Download all mock projects
+RUN git clone https://github.com/JetBrains-Research/plugin-utilies-mock-data /mock-data
```

Comment:
Are you sure about this url?  Should we replace with https://github.com/JetBrains-Research/plugin-utilies-mock-data.git?